lib/mongo-milestone-db.js

```diff
@@ -57,13 +57,21 @@ class MongoMilestoneDB extends MilestoneDB {
   }
 
   getMilestoneSnapshot(collectionName, id, version, callback) {
-    if (!collectionName) return process.nextTick(callback, new InvalidCollectionNameError());
-    if (!id) return process.nextTick(callback, new InvalidIdError());
-    if (!this._isValidVersion(version)) {
-      return process.nextTick(callback, new InvalidVersionError());
-    }
+    this._getMilestoneSnapshotByVersion(collectionName, id, version)
+      .then(snapshot => process.nextTick(callback, null, snapshot))
+      .catch(error => process.nextTick(callback, error));
+  }
+
+  getMilestoneSnapshotAtOrBeforeTime(collection, id, timestamp, callback) {
```

Comment:
Didn't see documentation for the `null` timestamp value specifically. Can we document in the readme the valid arguments? It appears that numbers and `null` are valid, but `undefined` is not. We should make it clear what null is supposed to do in this case in the README or code

Reply:
Do you want this in the `README` for this repo, or in `sharedb`? Because that's where the behaviour is defined, so maybe it makes the most sense to document it there?

Reply:
As discussed, I'll document in ShareDB because that's where it's defined.